auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -116,9 +118,12 @@ public ServerResponse create() {
         policy = policyDAO.find(requestBody.get("PolicyARN"));
       }
       catch (DataAccessException ex) {
+        LOGGER.error("Failed to find the requested policy in ldap");
         return responseGenerator.internalServerError();
       }
-      if (policy == null || !policy.exists()) {
```

Comment:
Can delete policy permission be granted to cross account using IAM Policy, 
Delete Policy is called with credentials and using credentials policy is searched in ldap, so there is no chance that account 1 can see policies of account, in that case why this check is required.

Reply:
Before deleting we search policy using ARN not by account. So account1 can try deleting policy of account2
Hence the check
In aws, cross account delete-policy not allowed.

Reply:
Fixed new design logic